modules/Bio/Otter/Lace/OnTheFly/QueryValidator.pm

```diff
@@ -132,6 +133,8 @@ sub _build_confirmed_seqs {     ## no critic (Subroutines::ProhibitUnusedPrivate
             $seq->sequence_string($s);
         }
     }
+    use Data::Dumper;
+    print Dumper(@confirmed_seqs);
```

Comment:
Please remove all instances of Data::Dumper. It occurs again at line 284. Are lines 285-286 required?